app/shared/requiresSignin.ts

```diff
@@ -0,0 +1,8 @@
+import * as pathLib from "path";
+
+const publicPaths = ["/contact-us-form"];
+
+export const requiresSignin = (path: string) => {
```

Comment:
Could percent-encoding `%2e%2e` maybe trick it? I guess it gets decoded by MMA before it reaches that point.

Reply:
Tested it locally and `%2e%2e` does not trick it. Seem like `normalize` takes care of it.

Reply:
`path.normalize` does not handle `%2e%2e`. If it was decoded then it must have happened higher up.

Reply:
Looked into it a bit closer and you are right. I used a jest test and the reason it doesn't trick it there is because `%2e%2e` is read as a string instead of being replaced by `..`. Tried to use it to access an authorised path from the browser but it also doesn't trick it. It seems like it is being decoded somewhere higher up as you said.